File: Frontend/angular-app/src/app/modules/admin/entities/audit.ts
import {AuditValueChanges} from "./audit-value-changes";

export class Audit {
  public id!: number;
  public userId!: number;
  public subSystem!: null;
  public menueId!: number;
  public type!: string;
  public tableName!: string;
  public transactionId!: string;
  public dateTime!: Date;
  public primaryId!: number;
  changesValuesCollection:AuditValueChanges [] = [];
}

File: Backend/Core/Eefa.Common/Data/Audit/Audit.cs
using System;
using System.ComponentModel.DataAnnotations.Schema;

namespace Eefa.Common.Data
{
    [Table("Audit", Schema = "dbo")]
    public class Audit:BaseEntity
    {
        public int UserId { get; set; }
        public string? SubSystem { get; set; }
        public int MenuId { get; set; }
        public string Type { get; set; }
        public string TableName { get; set; }
        public Guid TransactionId { get; set; }
        public DateTime DateTime { get; set; }
        public string ChangesValues { get; set; }
        public int PrimaryId { get; set; }
    }
}